docs/api/index.rst

```diff
@@ -21,6 +21,7 @@ The REST API Reference
     product_attributes
     product_options
     product_variants
+    product_reviews
```

Comment:
I guess that `product_reviews` should be before `product_variants` to keep alphabetical order